src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -5605,6 +5605,34 @@ public object Method(int i)
             await AssertFormatAsync(expectedCode, code);
         }
 
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/72196")]
+        public async Task FormatSwitchExpression_ListPatternAligned()
+        {
```

Comment:
can you please use 'raw string literals' for test cases.  There is a code refactoring to fix this up btw.